js/everything.js

```diff
@@ -498,17 +508,29 @@ function convertTimestamp(timestamp) {
     var hours = 0;
     var timeComponents = timestamp.split(":");
     if (timeComponents.length === 3) {
-        hours = parseInt(timeComponents[0], 10) * 60 * 60; // convert hours to seocnds
-        minutes = parseInt(timeComponents[1], 10) * 60; // convert minutes to seconds
-        seconds = parseInt(timeComponents[2], 10); // add remaining seconds
+        hours = convertHoursToSeconds(timeComponents[0]);
+        minutes = convertMinutesToSeconds(timeComponents[1]);
+        seconds = getSeconds(timeComponents[2]);
     }
     else {
-        minutes = parseInt(timeComponents[0], 10) * 60; // convert minutes to seconds
-        seconds = parseInt(timeComponents[1], 10); // add remaining seconds
+        minutes = convertMinutesToSeconds(timeComponents[0]);
+        seconds = getSeconds(timeComponents[1]);
     }
     return hours + minutes + seconds;
 }
 
+function convertHoursToSeconds(hours) {
+    return parseInt(hours, 10) * timeIntervals.SECONDS * timeIntervals.SECONDS;
+}
+
+function convertMinutesToSeconds(minutes) {
+    return parseInt(minutes, 10) * timeIntervals.SECONDS;
+}
+
+function getSeconds(seconds) {
```

Comment:
I'd like to rename this function to something like `parseBase10Int` - and use it in the above 2 functions for code reuse.